### PR TITLE
Prep for 2.6.0b1 Release

### PR DESCRIPTION
**Pull request by sydney-runkle**
* Updated `version.py`
* Updated `HISTORY.md`
* Pulled in the changes from v1.10.14

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -1,3 +1,125 @@
...
+* Introducing `classproperty` decorator for `model_computed_fields` by @Jocelyn-Gas in [#8437](https://github.com/pydantic/pydantic/pull/8437)
+* Explicitly raise an error if field names clashes with types by @Viicos in [#8243](https://github.com/pydantic/pydantic/pull/8243)
+* Use stricter serializer for unions of simple types by @alexdrydew [pydantic/pydantic-core#1132](https://github.com/pydantic/pydantic-core/pull/1132)
+
+#### Performance
+
+* Add Codspeed profiling Actions workflow  by @lambertsbennett in [#8054](https://github.com/pydantic/pydantic/pull/8054)
+* Improve `int` extraction by @samuelcolvin in [pydantic/pydantic-core#1155](https://github.com/pydantic/pydantic-core/pull/1155)
+* Improve performance of recursion guard by @samuelcolvin in [pydantic/pydantic-core#1156](https://github.com/pydantic/pydantic-core/pull/1156)
+* `dataclass` serialization speedups by @samuelcolvin in #1162 [pydantic/pydantic-core#1162](https://github.com/pydantic/pydantic-core/pull/1162)
```
Comment (samuelcolvin) [inline HISTORY.md:46]:
```suggestion
* `dataclass` serialization speedups by @samuelcolvin in [pydantic/pydantic-core#1162](https://github.com/pydantic/pydantic-core/pull/1162)
* Avoid `HashMap` creation when looking up small JSON objects in `LazyIndexMaps` by @samuelcolvin in [pydantic/jiter#55](https://github.com/pydantic/jiter/pull/55)
* use hashbrown to speedup python string caching by @davidhewitt in [pydantic/jiter#51](https://github.com/pydantic/jiter/pull/51)
* Replace `Peak` with more efficient `Peek` by @davidhewitt in [pydantic/jiter#48](https://github.com/pydantic/jiter/pull/48)
```

Comment (samuelcolvin) [reply HISTORY.md:46]: this is adding some performance things from jiter.

**Review by samuelcolvin — COMMENTED** [review]
otherwise LGTM.

File: HISTORY.md (hunk 1 above)
Comment (sydney-runkle) [reply HISTORY.md:46]: Great!

**Pull request merged**